src/containers/StructurePage/folderComponents/StructureErrorIcon.tsx

```diff
@@ -7,23 +7,22 @@
  */
 
 import { useTranslation } from "react-i18next";
-import styled from "@emotion/styled";
-import { spacing, colors } from "@ndla/core";
-import { AlertCircle } from "@ndla/icons/editor";
+import { ErrorWarningFill } from "@ndla/icons/common";
+import { styled } from "@ndla/styled-system/jsx";
 import { Node } from "@ndla/types-taxonomy";
 import { getIdFromUrn } from "../../../util/taxonomyHelpers";
 
-const StyledWarnIcon = styled(AlertCircle)`
-  height: ${spacing.nsmall};
-  width: ${spacing.nsmall};
-  fill: ${colors.support.red};
-`;
-
-const StyledAlertIcon = styled(AlertCircle)`
-  height: ${spacing.nsmall};
-  width: ${spacing.nsmall};
-  fill: ${colors.brand.grey};
-`;
+const StyledErrorWarningFill = styled(ErrorWarningFill, {
+  defaultVariants: {
+    variant: "warning",
+  },
+  variants: {
+    variant: {
+      warning: { fill: "icon.subtle" },
+      error: { fill: "surface.danger" },
```

Comment:
Tror jeg foretrekker `text.error` over `surface.danger`. Men skal sjekke om vi kan få en ny farge!

Reply:
Hedvig har brukt `surface.danger` i designet, derfor jeg gikk for den! men det aller beste hadde vært en icon.danger/icon.error